Update Soapbox project page to reflect current work (#626)
OpenSats/website: data/projects/soapbox.mdx

bridge to connect the fediverse to nostr. When OpenSats first funded the project
in 2023, [Ditto] was a nostr-native community server compatible with Mastodon
apps. Since then it has grown into a full organization building open-source
freedom tools across social media, AI-powered development, and activism, all on
nostr.

Since then, Ditto has evolved from a community server into a full social platform
where users can customize their profiles, publish any kind of content, and stay
connected to followers on Bluesky and Mastodon through built-in bridges.
Recently, Soapbox released [Shakespeare], an
open-source AI app builder that runs in the browser and works with any AI
provider. It was used to build Ditto 2.0 itself and has shipped as a standalone
product with its own Android app.

> I'm vegan btw.
>
> <cite>—[Alex Gleason]</cite>

Alex also maintains [Nostrify], a nostr framework for [Deno] and the web, and
[MKStack], a full-stack toolkit for building nostr apps with AI.

## Why fund it?

Bridging nostr and the fediverse expands the reach of both networks. The Mostr
bridge alone exposes millions of fediverse users to nostr, and several have

> already moved over entirely. Ditto brings the unique experiences of multiple different Nostr apps all under one roof, including Photos, Videos, Music, Podcasts, and more.

Reviewer: ```suggestion
already moved over entirely. Ditto brings the unique experiences of multiple different Nostr apps all under one roof, including photos, videos, music, podcasts, and more.
```

Shakespeare extends that further by letting anyone build new experiences on
nostr and deploy them without touching a terminal.

OpenSats has been supporting Soapbox since the [first wave of nostr
grants][wave-1] in July 2023. Alex later received a [long-term support
grant][lts] in August 2024.

## What's next?

> Soapbox is pushing the boundaries of nostr and making it the best and most

Reviewer: ```suggestion
```

Reviewer: Repetition

announcementLink: '/blog/nostr-grants-july-2023#soapbox'
---

Soapbox started as an alternative Mastodon frontend, then built the [Mostr]
bridge to connect the fediverse to nostr. When OpenSats first funded the project
in 2023, [Ditto] was a nostr-native community server compatible with Mastodon

> apps. Since then it has grown into a full organization building open-source

Reviewer: ```suggestion
Soapbox started as an alternative Mastodon frontend. It then moved on to build the [Mostr]
bridge,  which connects the fediverse to nostr, and [Ditto], a nostr-native community server compatible with Mastodon apps. 
Since then it has grown into a full organization building open-source
```

Author: Thank you for the feedback! 

re: "and [Ditto], a nostr-native community server compatible with Mastodon apps. " 

Ditto is not any longer "a community server compatible with Mastodon apps" (this was the old version from 2023). Today, Ditto is entirely a Nostr frontend application, which is why I reworded it this way. Sorry for the confusion!

Reviewer: ```suggestion
Soapbox started as an alternative Mastodon frontend. It then moved on to build the [Mostr]
bridge, which connects the fediverse to nostr, and [Ditto], which started as a nostr-native community server compatible with Mastodon apps.
Since then it has grown into a full organization building open-source
```

Reviewer: What about this? Does it make more sense? The following sentence highlights what Ditto has become.

Author: Yes, I think this makes more sense! Thank you!

freedom tools across social media, AI-powered development, and activism, all on
nostr.

[Ditto] has evolved from that community server into a full social platform

> where users customize their profiles, publish any kind of content, and stay

Reviewer: ```suggestion
Since then, Ditto has evolved from a community server into a full social platform
where users can customize their profiles, publish any kind of content, and stay
```

connected to followers on Bluesky and Mastodon through built-in bridges.
[Shakespeare] is an

> open-source AI app builder that runs in the browser and works with any AI

Reviewer: ```suggestion
Recently, Soapbox released [Shakespeare], an
open-source AI app builder that runs in the browser and works with any AI
```

provider. It was used to build Ditto 2.0 itself and has shipped as a standalone
product with its own Android app.

> I'm vegan btw.
>
> <cite>—[Alex Gleason]</cite>

Alex also maintains [Nostrify], a nostr framework for [Deno] and the web, and
[MKStack], a full-stack toolkit for building nostr apps with AI.

## Why fund it?

> Bridging nostr and the fediverse expands the reach of both networks. The [Mostr]

Reviewer: ```suggestion
Bridging nostr and the fediverse expands the reach of both networks. The Mostr
```

Reviewer: We should avoid using redundant links

bridge alone exposes millions of fediverse users to nostr, and several have
already moved over entirely. Ditto bridges across content types within nostr

> itself, bringing dozens of different micro experiences into a single platform.

Reviewer: Could you please rework this sentence? It is not completely clear to me.

Author: Rewording to: "Ditto brings the unique experiences of multiple different Nostr apps all under one roof, including Photos, Videos, Music, Podcasts, and more."

Shakespeare extends that further by letting anyone build new experiences on
nostr and deploy them without touching a terminal.

OpenSats has been supporting Soapbox since the [first wave of nostr
grants][wave-1] in July 2023. Alex later received a [long-term support
grant][lts] in August 2024.

## What's next?

Soapbox is pushing the boundaries of nostr and making it the best and most

> accessible place to build. Future plans include continuing to strengthen the

Reviewer: ```suggestion
Soapbox is pushing the boundaries of nostr to make it the best place for creators and communities
to build and thrive. Future plans include strengthening the
```

> connection between Ditto and Shakespeare, improving the ability to create and

Reviewer: ```suggestion
connection between Ditto and Shakespeare, enhancing the ability to create and
```

share social experiences, and improving the bridges between nostr and other
corners of the open web. The goal is to make nostr the fun and easy place for creators and communities
to build and thrive. People with no programming
experience are already using these tools to ship real applications, and that

> feedback loop between building and using is only getting tighter.

Reviewer: ```suggestion
corners of the open web.
```

> In 2026, Soapbox is teaming up with the World Liberty Congress to launch

Reviewer: ```suggestion
In 2026, Soapbox is teaming up with the [World Liberty Congress] to launch
```

[Agora], a spin-off of Ditto built with Shakespeare tailored to activist

> fundraising needs using Bitcoin. The team also plans to tackle Bitcoin

Reviewer: ```suggestion
[Agora], a bitcoin-based fundraising tool built with Shakespeare and tailored to activists'
needs. The team also plans to tackle Bitcoin
```

onboarding for general audiences, making sending sats to each other on nostr

> easier than ever with no Bitcoin knowledge required.

Reviewer: ```suggestion
onboarding for general audiences, making exchanging sats on nostr
easier than ever, even with no prior Bitcoin knowledge.
```

> [Agora]: https://soapbox.pub/agora/

Reviewer: Broken link

Author: Oops I will fix that one on our website! Good catch 🙏

[Alex Gleason]: https://njump.me/npub108pv4cg5ag52nq082kd5leu9ffrn2gdg6g4xdwatn73y36uzplmq9uyev6
[Deno]: https://deno.com/
[Ditto]: https://ditto.pub/

> [Google Play]: https://play.google.com/store/apps/details?id=pub.ditto.app

Reviewer: ```suggestion
```

Removing the link since it is not referenced anywhere.

[lts]: /blog/alex-gleason-receives-lts-grant
[MKStack]: https://gitlab.com/soapbox-pub/mkstack
[Mostr]: https://mostr.pub/
[Nostrify]: https://nostrify.dev/
[Shakespeare]: https://shakespeare.diy/
[wave-1]: /blog/nostr-grants-july-2023#soapbox

> [Zapstore]: https://zapstore.dev/apps/pub.ditto.app

Reviewer: ```suggestion
[World Liberty Congress]: https://worldlibertycongress.org/
```

Removed zapstore link (already present at the beginning) and added WLG one.
